modules/core/src/main/java/org/apache/ignite/internal/processors/security/NoOpIgniteSecurityProcessor.java

```diff
@@ -40,6 +41,9 @@
  * No operation IgniteSecurity.
  */
 public class NoOpIgniteSecurityProcessor extends GridProcessorAdapter implements IgniteSecurity {
+    /** Error message that occurs when trying to perform security operations if security is disabled. */
+    public static final String SECURITY_DISABLED_ERROR_MSG = "Operation cannot be performed: Ignite security is disabled.";
```

Comment:
typo: "Ignite security disabled"

Reply:
Is it proposed to remove "is" and write "Operation cannot be performed: Ignite security disabled"?

Reply:
Yes.

Reply:
Done.